docs/running-on-mesos.md

```diff
@@ -387,6 +387,13 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.coarse.shutdown.ms</code></td>
+  <td><code>10000</code> (10 seconds)</td>
+  <td> 
+    Time (in ms) to wait for executors to report that they have exited. Setting this too low has the risk of shutting down the Mesos driver (and thereby killing the spark executors) while the executor is still in the process of exiting cleanly.
```

Comment:
Please reconcile with https://github.com/apache/spark/pull/10319#issuecomment-172006944 on if this needs to be here